aspnetcore/mvc/models/model-binding.md

```diff
@@ -353,7 +353,7 @@ The `[Bind]` attribute can be used to protect against overposting in *create* sc
 ```csharp
 [HttpPost]
 public IActionResult OnPost(
-    [ModelBinder(typeof(MyInstructorModelBinder))] Instructor instructor)
+    [ModelBinder<MyInstructorModelBinder>] Instructor instructor)
```

Comment:
Do we need to add a guard here so we only render the generic attribute for docs targeting 8.0 and above?